api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -64,7 +64,6 @@
 import org.pmiops.workbench.model.UsernameTakenResponse;
 import org.pmiops.workbench.moodle.ApiException;
 import org.pmiops.workbench.compliance.ComplianceService;
```

Comment:
ComplianceService can be ripped out of this class now, right? Or am I missing somewhere it's used?

Reply:
Yes, it should not be ref here now. Have updated the file